code/LogFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SilverStripe\Freshdesk;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\SyslogHandler;
+use Monolog\Logger;
+use Monolog\Processor\WebProcessor;
+use SilverStripe\Framework\Injector\Factory;
+
+class LogFactory implements Factory
+{
+    public function create($service, array $params = [])
```

Comment:
[Apparently](https://scrutinizer-ci.com/g/silverstripeltd/silverstripe-freshdesk/inspections/0df73f19-e3fb-4c41-99f6-044b318dd0e0/issues/files/code/LogFactory.php?status=new&orderField=path&order=asc&fileId=code%2Fcontrol%2FFreshdeskSsoController.php&honorSelectedPaths=0) you are not supposed to return non SS_Object here?